source/nodejs/adaptivecards-designer/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "adaptivecards-designer",
-	"version": "2.5.0-alpha.0",
+	"version": "2.4.3",
```

Comment:
Why did we change this?

Reply:
@anna-dingler this is the value used in npm js

Reply:
I thought we switched to the alpha annotation for lerna bootstrap?

Reply:
all right, thanks, i will fix it.